internal/pkg/plugin/trello/create.go

```diff
@@ -1,27 +1,32 @@
 package trello
 
 import (
+	"fmt"
+
+	"github.com/mitchellh/mapstructure"
+
 	"github.com/merico-dev/stream/pkg/util/log"
 )
 
 // Create creates Tello board and lists(todo/doing/done).
 func Create(options map[string]interface{}) (map[string]interface{}, error) {
-	var opts *Options
-	var err error
-
-	if opts, err = convertMap2Options(options); err != nil {
+	var opts Options
+	if err := mapstructure.Decode(options, &opts); err != nil {
```

Comment:
Maybe we can add it into some document to let future developers know which function to use to decode options.

Reply:
Get it